domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -1023,16 +1074,16 @@ public void saveBatchNullConnection(TestContext context) {
 
   @Test
   public void saveBatchNullList(TestContext context) {
-    createFoo(context).saveBatch(FOO, (List<Object>)null, context.asyncAssertSuccess(save -> {
-      context.assertEquals(0, save.getNumRows());
+    createFoo(context).saveBatch(BAR, (List<Object>)null, context.asyncAssertSuccess(save -> {
+      context.assertNull(save);
     }));
   }
 
   @Test
   public void saveBatchEmptyList(TestContext context) {
     List<Object> list = Collections.emptyList();
     createFoo(context).saveBatch(FOO, list, context.asyncAssertSuccess(save -> {
-      context.assertEquals(0, save.getNumRows());
+      context.assertNull(save);
```

Comment:
This is an unneccessary breaking change that is inconsistent with vertx-pg-client that returns an empty RowSet when "UPDATE ... WHERE ..." does not update any record.
Can you return an empty RowSet?

Reply:
Yes